CMSIS-SVD/Cortex_M/cm0.svd

```diff
@@ -0,0 +1,928 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  Copyright (C) 2016 AdaCore
+ -->
+
+<device schemaVersion="1.2" xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="CMSIS-SVD.xsd" >
+  <peripherals>
+    <peripheral>
+      <name>NVIC</name>
+      <baseAddress>0xE000E100</baseAddress>
+      <access>read-write</access>
+
+      <addressBlock>
+        <offset>0</offset>
+        <size>0x450</size>
+        <usage>registers</usage>
+      </addressBlock>
+
+      <registers>
+        <register>
+          <name>NVIC_ISER</name>
+          <description>Interrupt Set-Enable Registers</description>
+          <addressOffset>0x00</addressOffset>
+          <size>32</size>
+          <access>read-only</access>
+        </register>
+        <register>
+          <name>NVIC_ICER</name>
+          <description>Interrupt Clear-Enable Registers</description>
+          <addressOffset>0x80</addressOffset>
+          <size>32</size>
+          <access>read-only</access>
+        </register>
+        <register>
+          <name>NVIC_ISPR</name>
+          <description>Interrupt Set-Pending Registers</description>
+          <addressOffset>0x100</addressOffset>
+          <size>32</size>
+          <access>read-only</access>
+        </register>
+        <register>
+          <name>NVIC_ICPR</name>
+          <description>Interrupt Clear-Pending Registers</description>
+          <addressOffset>0x180</addressOffset>
+          <size>32</size>
+          <access>read-only</access>
+        </register>
+        <register>
+          <name>NVIC_IPR</name>
+          <description>Interrupt Priority Register</description>
+          <addressOffset>0x300</addressOffset>
+          <size>8</size>
```

Comment:
the IPR registers are only word-accessible, so should be binded to a 32-bit register.